lib/vrl/stdlib/benches/benches.rs

```diff
@@ -1734,6 +1735,20 @@ bench_function! {
     }
 }
 
+bench_function! {
+    reverse_dns => vrl_stdlib::ReverseDns;
+
+    localhost {
+        args: func_args![value: value!("127.0.0.1")],
+        want: Ok(value!("localhost")),
+    }
+
+    google {
```

Comment:
This could be fairly brittle as it relies on a number of external factors. It's probably important to bench this though, so I think we should keep it.

I wonder if `8.8.8.8` is the best to use since a lot of systems may be setup to use it for DNS so it may already be cached by the OS.

Reply:
Yeah, true, though these DNS resolvers IPs also seem to be the least likely to change 😄  Also, if the OS is caching these lookups, it'll be cached after the first lookup anyway?